README.md

```diff
@@ -24,22 +24,97 @@ iex(1)>
 Extending for your own application:
 
 ```dockerfile
-FROM bitwalker/alpine-elixir:1.9.4
+#https://github.com/bitwalker/alpine-elixir/issues/51
```

Comment:
Now that #51 is closed, we can swap back to the explicit version here, then I think we're good to go!